packages/pass-style/src/remotable.js

```diff
@@ -38,39 +47,48 @@ export const canBeMethod = func =>
 harden(canBeMethod);
 
 /**
- * Abstract out this test so a later PR can restrict possible method names
+ * Is `key` a possible method name of a remotable? To qualify, `key` must be
+ * a string other than `then` or `constructor`.
+ *
+ * TODO How we bind an OCapN symbol "then" or "constructor" to JS remotables
+ * is still TBD. Probably some form of Hilbert hotel.
  *
  * @param {any} key
  * @returns {key is RemotableMethodName}
  */
 const canBeMethodName = key =>
```

Comment:
Do the whole check/is/assert pattern, and consolidate new error message in `checkMethodName`.

What should this do with `Symbol.asyncIterator` and `@@asyncIterator`?

Reply:
Not in this PR